src/components/Instances/InstanceDetail/InstanceDetail.js

```diff
@@ -18,7 +21,11 @@ const InstanceDetail = ({ description, rootGroupId, groups, isAdmin }) =>
           />
         }
       >
-        <ReactMarkdown source={description} />
+        <div
+          dangerouslySetInnerHTML={{
+            __html: md.render(description)
+          }}
+        />
```

Comment:
This raw-html-thing is rather ugly and repeats itself in other files. Further more, initialization of markdown-it also requires special handling. Would you mind wrapping it up in a component?